core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -424,6 +426,13 @@ case object KAFKA_2_7_IV2 extends DefaultApiVersion {
   val id: Int = 30
 }
 
+case object KAFKA_2_7_IV3 extends DefaultApiVersion {
```

Comment:
2.7 has been branched. It should be 2.8, right?

Reply:
Thanks, that makes more sense.